src/java.base/share/classes/sun/util/resources/CurrencyNames.properties

```diff
@@ -512,5 +513,6 @@ yum=Yugoslavian New Dinar (1994-2002)
 zar=South African Rand
 zmk=Zambian Kwacha
 zwd=Zimbabwean Dollar (1980-2008)
+zwg=Zimbabwe Gold
```

Comment:
It's interesting to see the difference `Zimbabwe` and `Zimbabwean` here. The latter is from CLDR and the name for ZWG is not yet available, so we may need to pull the name from CLDR later.

Reply:
I see your point. CLDR takes precedence over ISO4217 regarding currencies here?

Reply:
That is more consistent IMO

Reply:
CLDR is aware of this change but hasn't added it yet. We're mid cycle so it won't be much more than metadata this round due to timing.  https://unicode-org.atlassian.net/issues/CLDR-17751 if you want to track.

Reply:
> CLDR is aware of this change but hasn't added it yet. We're mid cycle so it won't be much more than metadata this round due to timing. https://unicode-org.atlassian.net/issues/CLDR-17751 if you want to track.

Out of curiosity, what's the rationale behind the currency name change from "Zimbabwe Dollar" (ISO 4217 definition) to "Zimbabwean Dollar"?

Reply:
> Out of curiosity, what's the rationale behind the currency name change from "Zimbabwe Dollar" (ISO 4217 definition) to "Zimbabwean Dollar"?

I'm not sure.